lib/reporters.js

```diff
@@ -47,7 +47,7 @@ class DatadogReporter {
             // URL from their web browser. More details on correct configuration:
             // https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site
             this.site = this.site.replace(/^app\./i, '');
-            datadogApiClient.client.setServerVariables(configuration, {
+            configuration.setServerVariables(configuration, {
```

Comment:
Can you check this PR? https://github.com/dbader/node-datadog-metrics/pull/113

I think that the first argument is not needed. Can we confirm this?

Reply:
Yeah, this code is not correct, and the first argument (`configuration`) should be removed. See also the actual docs for [the old, deprecated method](https://datadoghq.dev/datadog-api-client-typescript/functions/client.setServerVariables.html) vs. [the new one you are using here](https://datadoghq.dev/datadog-api-client-typescript/classes/client.Configuration.html#setServerVariables).

Reply:
I can reopen https://github.com/dbader/node-datadog-metrics/pull/113 if you want

Reply:
If you want, that's fine. There's no actual emergency here, though. You'd also need to make the `package.json` change there.